src/main/java/core/mate/academy/service/BulldozerProducer.java

```diff
@@ -0,0 +1,13 @@
+package core.mate.academy.service;
+
+import core.mate.academy.model.Bulldozer;
+import java.util.List;
+
+public class BulldozerProducer implements MachineProducer<Bulldozer> {
+    @Override
+    public List<Bulldozer> get() {
+        return List.of(new Bulldozer("t7000"),
+                new Bulldozer("t3000"),
+                new Bulldozer("t5000"));
```

Comment:
better use this format
```suggestion
        return List.of(
            new Bulldozer("t7000"),
            new Bulldozer("t3000"),
            new Bulldozer("t5000")
        );
```